qutebrowser/mainwindow/prompt.py

```diff
@@ -633,6 +650,35 @@ def __init__(self, question, parent=None):
         self._to_complete = ''
         self._root_index = QModelIndex()
 
+        self._history = cmdhistory.History(parent=self)
+        fprompt_history = objreg.get('fprompt-history')
+        self._history.history = fprompt_history.data
+        self._history.changed.connect(fprompt_history.changed)
+
+    def history_prev(self):
+        """Go back in the history."""
+        try:
+            if not self._history.is_browsing():
+                item = self._history.start(text="")
+            else:
+                item = self._history.previtem()
+        except (cmdhistory.HistoryEmptyError,
+                cmdhistory.HistoryEndReachedError) as e:
+            print(e)
+            print("error")
```

Comment:
This probably shouldn't be here.

Reply:
Yeah I really need to remember to remove my debugging print statements. Maybe I can find a VSCode extension that highlights all print statements.